cmd/lotus-shed/dagspliter/dagsplitter.go

```diff
@@ -298,3 +325,10 @@ var Cmd = &cli.Command{
 		return nil
 	},
 }
+
+// FIXME: Add real test. For now check that the output matches across refactors.
```

Comment:
A (manual) test I'm using is adding the rather large pond node_modules directory (`make lotus-pond-app; ipfs add -r lotuspond`), then chunking with 31M chunk size, and then making sure that no resulting car files are above 32M

Reply:
That's valid but I was more trying to make sure I don't change the output at all during some parts of the refactoring (matching CIDs apart from size restriction).